sklearn/neural_network/tests/test_mlp.py

```diff
@@ -267,6 +267,33 @@ def test_lbfgs_regression():
             assert_greater(mlp.score(X, y), 0.95)
 
 
+def test_lbfgs_maxfun():
+    # Test lbfgs parameter max_fun.
+    # It should independently limit the number of iterations for lbfgs.
+    max_fun = 10
+    # classification tests
+    for X, y in classification_datasets:
```

Comment:
please move these to `@pytest.mark.parameterize`, it makes it easier to debug test failures.

Reply:
Happy to do this, but I don't know how.  Can you provide more guidance?

Reply:
For example, does this mean I just need to add an annotation somewhere?
Do I need to move the test function body to a different file?
Do I need to just copy the name of this new test function somewhere?
Thanks for any additional details/guidance.

Reply:
@adrinjalali  - I read up on the @pytest.mark.parameterize functionality. I moved the dataset iteration into the @pytest.mark.parameterize annotation as per your comments.  Please let me know if this addresses your concern.  If so please do resolve the conversation and approve the PR.  Thanks.